Bottom: eb3b6bb07d110ffb8d3f46299f1ffb4a9acce058
Top:    794a7ccd18fea22a0006c5b1e8a9c8d11e7e13b3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-22 02:55:12 +0000

Refresh of butcher-the-xor-crosshair-draw

---

diff --git a/src/draw_api.h b/src/draw_api.h
index 69091c8..7fadd9f 100644
--- a/src/draw_api.h
+++ b/src/draw_api.h
@@ -54,7 +54,7 @@ struct DrawAPI {
 
   /* Member variables */
 //  GraphicsAPI *graphics;
-  HID_DRAW_API *graphics;
+  HID_DRAW *graphics;
   hidGC gc;
   hidGC fg_gc;
   hidGC bg_gc;
